### PR TITLE
public to private

### PR DESCRIPTION
**Pull request by ihoment-meihui**
*Issue #311 *

*Description of changes:*
change the description on line 85 for testing without using private key.
old: "device is instantiated with no public key"
new: "device is instantiated with no private key"

### CODE REVIEW

File: test/device-unit-tests.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ describe( "device class unit tests", function() {
             ); 
       });
    });
-   describe( "device is instantiated with no public key", function() {
+   describe( "device is instantiated with no private key", function() {
 //
 // Verify that the device module throws an exception when there is
 // no valid public key file.
```
Comment (jmklix) [inline test/device-unit-tests.js:88]: Can you update the comment (public to private) as well?

Comment (ihoment-meihui) [reply test/device-unit-tests.js:88]: OK

Comment (jmklix) [reply test/device-unit-tests.js:88]: Just to clarify I meant the comment on line 88
 `// no valid public key file.`
 change to 
 `// no valid private key file.`

**Comment by bretambrose** [comment]
Merged via https://github.com/aws/aws-iot-device-sdk-js/pull/379

**Pull request closed without merging**